keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -656,8 +690,8 @@ feature gate controls overall availability of the feature.
 
 ### Version Skew Strategy
```

Comment:
In the `### Upgrade / Downgrade Strategy` section, I think we need to cover how to migrate away from cluster trust bundles used in an app.

Reply:
This might be considered covered by some of the later sections?

Reply:
To clarify, you are referring to migrating an individual application from reading root certs from a bundled file (or whatever) onto reading a projected ClusterTrustBundle?